server/config/config.json

```diff
@@ -7,17 +7,11 @@
     "dialect": "postgres"
   },
   "test": {
-    "username": "root",
-    "password": null,
-    "database": "database_test",
-    "host": "127.0.0.1",
+    "use_env_variable": "D_MAN",
     "dialect": "postgres"
   },
   "production": {
-    "username": "root",
-    "password": null,
-    "database": "database_production",
-    "host": "127.0.0.1",
+    "use_env_variable": "D_MAN",
```

Comment:
*Nitpick*
  I would rather use something like DB_URL or related for ease of understanding to other developers.